routes/api/v1/users.js

```diff
@@ -16,6 +27,11 @@ router.post( '/', (request, response ) => {
     .catch( errorResponse( response ))
 })
 
+router.get( '/logout', ( request, response ) => {
+  request.logout()
+  response.send('User has logged out')
```

Comment:
We should not be sending strings from the API, we should be sending a status code.